storage/rocksdb/ha_rockspart.cc

```diff
@@ -97,6 +88,13 @@ handler *ha_rockspart::clone(const char *name, MEM_ROOT *mem_root) {
             (uchar *)alloc_root(mem_root, ALIGN_SIZE(ref_length) * 2)))
     goto err;
 
+  /* We will not use clone() interface to clone individual partition
```

Comment:
With m_is_clone_of gone, is this comment still relevant? Isn't the current implementation the natural one now?

Reply:
Well, the natural instinct to clone ha_tokupart/rockspart/Partition_base::clone() is to again use clone() interface for individual partition handlers.

If it is not confusing, I would suggest to keep as is. But am OK with removal as well

Reply:
OK, let's keep it